pkg/apis/triggers/v1alpha1/event_listener_validation.go

```diff
@@ -27,8 +27,8 @@ func (t *EventListener) Validate(ctx context.Context) *apis.FieldError {
 }
 
 func (s *EventListenerSpec) Validate(ctx context.Context) *apis.FieldError {
-	if len(s.TriggerBindingRefs) == 0 {
-		return apis.ErrMissingField("spec.triggerbindingrefs")
+	if len(s.Triggers) == 0 {
+		return apis.ErrMissingField("spec.triggers")
```

Comment:
should we validate the contents of the `triggers` also?

Reply:
There is a separate issue open for validation so I didn't opt to tackle it here just yet. Also, since we are still in a bit of flux, didn't want this to be cumbersome.

Reply:
kk sounds good!

Reply:
Resolving since separate issue